src/data/prisma.ts

```diff
@@ -0,0 +1,30 @@
+import { PrismaClient } from "@prisma/client";
```

Comment:
Me trying to figure out how the heck to not have some many connection processes flying around.

Reply:
So did this end up solving the problem you noted above?

> some weird things where there are a ton of processes managing database connections